explainer.md

```diff
@@ -325,45 +320,46 @@ If a site wishes to recommend a TV series to a user they can use the [TVSeries](
 
 ### A site recommends some live video content to a user
 
-If a site wishes to recommend some live video content to a user they can add a [BroadcastEvent](https://schema.org/BroadcastEvent) in the publication property (this is also on the TVEpisode and Movie media item types as well). The event should have the `isLiveBroadcast` property set to true and the `startDate` property set to the time the broadcast started. The `endDate` should be included too if known.
+If a site wishes to recommend a generic video to a user they can add a [BroadcastEvent](https://schema.org/BroadcastEvent) to the feed. This is a special type that describes a broadcast schedule or a live broadcast and embeds the media item being broadcast in the `workPerformed` property.
```

Comment:
I don't understand what "generic video" means here. It's not actually generic compared to the other types of content, right? It's just a specific type. I like the description in the second sentence, but do you think you could reword this first part somehow?

Reply:
Thanks for the rewording. Looks better to me.